multicluster/cmd/multicluster-controller/options.go

```diff
@@ -41,12 +45,31 @@ func (o *Options) complete(args []string) error {
 	var err error
 	options := ctrl.Options{Scheme: scheme}
 	ctrlConfig := &mcsv1alpha1.MultiClusterConfig{}
+	klog.Info("Config file is ", o.configFile)
 	if len(o.configFile) > 0 {
 		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(o.configFile).OfKind(ctrlConfig))
 		if err != nil {
 			return fmt.Errorf("fail to load options from configuration file %s", o.configFile)
 		}
 		o.options = options
+
+		data, err := ioutil.ReadFile(o.configFile)
+		if err != nil {
+			return err
+		}
+		err = yaml.Unmarshal(data, &ctrlConfig)
+		if err != nil {
+			return err
+		}
+		o.leader = ctrlConfig.Leader
```

Comment:
`options` at L50 does not load these options? 

if not, then i would suggest a new method  `o.loadConfigFromFile()`

Reply:
Sure, added a todo to resolve this in the next patch